controllers/apply-next/views/step.njk

```diff
@@ -35,26 +35,27 @@
                 {{ formErrors(errors = errors) }}
 
                 {% for fieldset in step.fieldsets %}
-                    <fieldset class="form-fieldset u-constrained-wide">
-                        <legend class="form-fieldset__legend t2 t--underline">
-                            {{ fieldset.legend }}
-                        </legend>
+                    {% if fieldset.fields.length > 0 %}
```

Comment:
This is whitespace heavy, but I'm wrapping this in a `fieldset.fields.length > 0` check to avoid rendering fieldsets with no fields in them. Maybe I should be filtering this out in `enhanceForm` rather than in the view?

Reply:
Hmm... possibly? There are a few places where we loop over the entire form (summary, email, new questions endpoint)... would be neater not to have to repeat this check multiple times.

Reply:
👍 I've moved this up into `enhanceForm` now